sessionctx/variable/session.go

```diff
@@ -367,6 +367,9 @@ type SessionVars struct {
 	Users map[string]types.Datum
 	// systems variables, don't modify it directly, use GetSystemVar/SetSystemVar method.
 	systems map[string]string
+	// sqlLevels variables are temporarily set by SET_VAR hint
+	// It only take effect for the duration of a single statement
+	sqlLevels map[string]string
```

Comment:
`sqlLevels` may make people confused about the actual usage. How about `updatableHints`?

Reply:
We have changed the variable name multiple times, let's discuss them here and make an agreement before changing them again. @qw4990 @b41sh

Reply:
How about `stmtVars` or `stmtScope`? I think how they are updated is inconsequential, and other scoped vars have multiple ways of being updated.

Reply:
`stmtVars` LGTM. So the variable and function name can be renamed to

- `stmtVars`
- `SetStmtVar()`

Reply:
I prefer `stmtVars` which has the same style as other variables like `globalVars`, `sessionVars`. @zz-jason @nullnotnil @b41sh

Reply:
ok, let's rename the variable and function proposed in https://github.com/pingcap/tidb/pull/20232#discussion_r511732145

Reply:
ok, I will change it.

Reply:
done